src/main/resources/templates/courseFormView.twig

```diff
@@ -1,32 +1,30 @@
 {% if courseCreationError %}
+    <script type="text/javascript" src="/static/js/jquery-3.2.1.min.js"></script>
+    <script type="text/javascript" src="/static/js/jquery-ui.min.js"></script>
+    <script type="text/javascript" src="/static/js/bootstrap.min.js"></script>
+    <script type="text/javascript" src="/static/js/mdb.min.js"></script>
     <script>
         toastr.options.timeOut = 0;
         toastr.error('{{ courseCreationError }}');
     </script>
 {% endif %}
 
-<div style="width:50%; margin: auto; padding-bottom: 60px;" class="collapse" id="courseForm">
-    <form action="/course" method="post">
+<div style="width:50%; margin: auto; padding-bottom: 60px;" class="{% if not courseCreationError %}collapse{% endif %}" id="courseForm">
+    <form action="/course" name="courseForm" method="post">
         <div class="md-form">
-            <input placeholder="Course Name" type="text" name="name" class="form-control" id="courseFormName"{% if courseCreationError %}value="{{ courseName }}"{% endif %}>
+            <input placeholder="Course Name (Required)" type="text" name="name" class="form-control" id="courseFormName"{% if courseCreationError %}value="{{ courseName }}"{% endif %}>
         </div>
         <div class="md-form">
             <input placeholder="Section" type="text" name="section" class="form-control" {% if courseCreationError %}value="{{ sectionName }}"{% endif %}>
         </div>
         <div class="md-form">
-            <input placeholder="Timeslot" type="text" name="timeSlot" class="form-control timepicker">
+            <input placeholder="Start Date (Required)" name="startDate" type="text" class="form-control datepicker" {% if courseCreationError %}value="{{ startDate }}"{% endif %}>
         </div>
         <div class="md-form">
-            <input placeholder="Start Date" name="startDate" type="text" class="form-control datepicker" >
+            <input id="course-form-rows" placeholder="Rows (Required)" name="rows" type="number" class="form-control" {% if courseCreationError %}value="{{ rowsAmount }}"{% endif %}>
         </div>
         <div class="md-form">
-            <input placeholder="End Date" name="endDate" type="text" class="form-control datepicker" >
-        </div>
-        <div class="md-form">
-            <input id="course-form-rows" placeholder="Rows" name="rows" type="number" class="form-control" {% if courseCreationError %}value="{{ rowsAmount }}"{% endif %}>
-        </div>
-        <div class="md-form">
-            <input id="course-form-cols" placeholder="Columns" name="cols" type="number" class="form-control" {% if courseCreationError %}value="{{ colsAmount }}"{% endif %}>
+            <input id="course-form-cols" placeholder="Columns (Required)" name="cols" type="number" class="form-control" {% if courseCreationError %}value="{{ colsAmount }}"{% endif %}>
```

Comment:
What if we have an (optional) tag for section instead of 4 (required) tags?

Reply:
I like this idea